example/build.gradle

```diff
@@ -47,4 +48,25 @@ dependencies {
     reactor_3_3_x 'io.projectreactor:reactor-core:3.3.0.BUILD-SNAPSHOT', {
         changing = true
     }
+
+    jmh project(path: ":agent", configuration: 'shadow')
+    //Uncomment for benchmark execution by Idea plugin
+//    jmh "org.openjdk.jmh:jmh-generator-annprocess:1.21"
+}
+
+jmh {
+    if (project.hasProperty("jmh.simpleTravisMode")) { //temporary for JHM sanity testing in Travis
+        include = 'reactor\\.blockhound\\.BlockHoundDummyBenchmark.*'
+        warmupIterations = 1
+        iterations = 3
+        fork = 1
+    } else {
+        include = 'reactor\\.blockhound\\.BlockHound.*Benchmark.*'
+        warmupIterations = 3
```

Comment:
I would prefer if there are configured as an annotation on the concrete test, because different benchmarks may use different parameters

Reply:
Currently it's not a case, but I removed that from `build.gradle`.